### DISCUSSION TITLE
Core-shell structure

### DISCUSSION THREAD

**Discussion by dharmabas** [neperfepx/neper#115] [Q&A] [answered]
Hi, 
Is there a way to make a core-shell structure in neper? Something like a sphere inside each cell. 

Help appreciated. Thank you.

**Comment by rquey** [comment, marked as answer]
Currently, no, although in principle it would not be too difficult to do (but it's not planned). Would you like to mesh this structure?

**Reply by dharmabas** [reply]
Yeah that's what I am looking for. And eventually assign block ids to each core-shell too.

**Reply by rquey** [reply]
Okay. So, Neper could be your friend...
